.github/workflows/maven-publish-android.yml

```
@@ -12,6 +12,11 @@ on:
         required: true
         default: "android"
         type: string
+    inputs:
```

actionlint (1.7.12)

[error] key "inputs" is duplicated in "workflow_call" section. previously defined at line:5,col:5

(syntax-check)